js/cedar.js

```diff
@@ -69,8 +65,8 @@ Cedar.Application = function(options) {
   Cedar.config.wait = this.options.wait;
   Cedar.config.fetch = this.options.fetch;
 
-  if (typeof Cedar.$ === "undefined") {
-    Cedar.$ = jQuery({});
+  if (Cedar.events === undefined) {
```

Comment:
That and in older browsers doing something like `if (document.getElementById === undefined)` would give you an error if that method didn't exist. That's how I got into the habit of doing it.